app/components/avo/fields/country_field/index_component.html.erb

```diff
@@ -1,7 +1,12 @@
 <%= index_field_wrapper **field_wrapper_args do %>
-  <% if @field.display_code %>
-    <%= @field.value %>
-  <% else %>
-    <%= @field.countries[@field.value] %>
-  <% end %>
+  <div class="flex flex-row items-center justify-between space-x-1 group">
+    <% if @field.display_code %>
+      <%= @field.value %>
+    <% else %>
+      <%= @field.countries[@field.value] %>
+    <% end %>
+    <% if copyable? %>
+      <%= render Avo::ClipboardComponent.new value: field_value %>
+    <% end %>
```

Comment:
Let's explore an alternative approach: render this once on the `index_field_wrapper` instead of manually adding it to each field we want to support. This would significantly reduce complexity.

Let's consider applying this approach to both `index_field_wrapper` and `field_wrapper`.